src/release/rollups.md

```diff
@@ -0,0 +1,75 @@
+# Rollup Procedure
+
+## Background
+
+The Rust project has a policy that every pull request must be tested after merge
+before it can be pushed to master. As PR volume increases this can scale poorly,
+especially given the long (~3.5hr) current CI duration for Rust.
+
+Enter rollups - changes that small, not performance sensitive, or not platform
+dependent are marked with the `rollup` command to bors (`@bors r+ rollup` to
+approve a PR and mark as a rollup, `@bors rollup` to mark a previously approved
+PR, `@bors rollup-` to un-mark as a rollup).  'Performing a Rollup' then means
+collecting these changes into one PR and merging them all at once. The rollup
+command accepts three values `always`, `maybe`, and `never`. `@bors rollup` is
+equivalent to `rollup=always` (which will always put a PR in a rollup), and
```

Comment:
Note: these commands, aside from `rollup` (which will make the priority effectively negative) don't actually have any automatic effect like that; it is a suggestion to the person making a rollup.

Reply:
Reworded to make that distinction more clear.